src/main/resources/static/scripts/lobby.js

```diff
@@ -24,37 +22,54 @@ let numberOfRows = Math.ceil(characterImgList.length / 5)
 wrapper.height(numberOfRows * characterSelectorScreen.width() / 5)
 characterSelectorScreen.css('grid-template', `repeat(${numberOfRows}, auto) / repeat(5, auto)`)
 
-
 let divs = []
 characterImgList.forEach((img,index) => {
 	let characterImg = document.createElement('div')
-	characterImg.style.backgroundImage = (`url(images/${img})`)
-	characterImg.id = img.split(".")[0]
+	characterImg.style.backgroundImage = (`url(${img})`)
+	characterImg.id = img.split("/").pop().split('.')[0]
 	characterImg.classList.add('characterImg')
 	characterSelectorScreen.append(characterImg)
 	divs.push(characterImg)
 })
 
 let selectors = []
+let zIndexes = [
+	[4,1,3,2],
+	[1,3,2,4],
+	[3,2,4,1],
+	[2,4,1,3]
+]
 
 Object.keys(players).forEach((_,index) => {
 	let selector = document.createElement('div')
-	selector.style.width = (divs[0].offsetWidth - 10)+ 'px'
-	selector.style.height = (divs[0].offsetHeight - 10) + 'px'
+	selector.style.width = (divs[0].offsetWidth)+ 'px'
+	selector.style.height = (divs[0].offsetHeight) + 'px'
 	selector.style.position = 'absolute'
-	selector.style.border = `solid 5px ${playersColors[index]}`
 	selector.classList.add('selector')
+	selector.style.display = 'grid'
+	selector.style.gridTemplate = 'auto auto / auto auto'
+
+	let parts = ['Top-Left','Top-Right','Bottom-Left','Bottom-Right']
+
+	parts.forEach((part,_index) => {
+		let selectorPart = document.createElement('div')
+		selectorPart.style.zIndex = zIndexes[_index][index]
+		part.split('-').forEach(borderDirection => {
+			selectorPart.style[`border${borderDirection}`] = `solid 10px ${playersColors[index]}`
+		})
+		selector.append(selectorPart)
+	})
+
 	selector.id = `player${index}`
 	characterSelectorScreen.append(selector)
 	selectors.push(selector)
 })
 
 let inMove = false
```

Comment:
"inMove" is close to being syntactically incorrect
"isMoving" is more appropriate